templates/terraform/nested_query.go.erb

```diff
@@ -0,0 +1,54 @@
+	var v interface{}
+	var ok bool
+
+<% object.nested_query.keys[0...-1].each do |k| -%>
+	v, ok = res["<%=k-%>"]
+	if !ok || v == nil {
+		return nil, nil
+	}
+	res = v.(map[string]interface{})
+
+<% end -%>
+	v, ok = res["<%=object.nested_query.keys[-1]-%>"]
+	if !ok || v == nil {
+		return nil, nil
+	}
+
+<%# Final nested resource is either a list of resources we need to filter
```

Comment:
When reading the downstream it was unclear what was happening here (https://github.com/terraform-providers/terraform-provider-google/pull/3259/files#diff-f287a8414c2c62fd8641384b31bae3e2R296), do you mind making this a Go comment instead?

Imo Ruby comments should describe unusual ruby code / concepts mm can't express well, and Go comments describe the mechanics of the outputted code.

Reply:
yeahh i switched it to ruby like two commits ago because it looked messy, I'll switch it back